02_03_hw2 NEW/css/style.css

```diff
@@ -0,0 +1,191 @@
+            box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+    background-color: #181818;
+    z-index: 999999;
+    overflow: hidden;
+}
+
+.header__title {
+    color: #767676;
+    font-family: "Proxima Nova Rg";
+    font-size: 15px;
+    font-weight: 400;
+    font-style: normal;
+    text-align: left;
+    text-transform: uppercase;
+    letter-spacing: 4.8px;
+    line-height: normal;
+}
+
+.header__text {
+    color: #fefefe;
+    font-family: "Playfair Display";
+    font-size: 48px;
+    font-weight: 400;
+    font-style: normal;
+    letter-spacing: normal;
+    line-height: normal;
+    text-align: left;
+}
+
+.cursor {
+    color: #edeff2;
+    opacity: 0.71;
+}
+
+.header__text::after {
+    content: "";
+    display: block;
+    width: 854px;
+    height: 3px;
+    background-color: #ffffff;
+    opacity: 0.1;
+    margin-top: 10px;
+}
+
+/* button */
+
+.button {
+    width: 100%;
+    max-width: 812px;
+    height: 76px;
+    -webkit-box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+            box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+    background-color: #fbc02d;
+    list-style-type: none;
+    position: relative;
+    top: 47px;
+    left: 484px;
+    margin: 0;
+    padding: 0;
+    z-index: 999999;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    font-size: 32px;
+}
+
+.button__icon {
+    color: white;
+    opacity: 0.5;
+    padding: 0 20px;
+}
+
+.button__icon:nth-child(1) {
+    margin-left: 20px;
+}
+
+.button__icon:nth-child(4) {
+    opacity: 1;
+}
+
+/* intro */
+.intro {
+    position: relative;
+    top: -110px;
+    width: 100%;
+    max-width: 1060px;
+    -webkit-box-shadow: 0 0 35px rgba(0, 0, 0, 0.22);
+            box-shadow: 0 0 35px rgba(0, 0, 0, 0.22);
+    background-color: #ffffff;
+    left: 180px;
+}
+
+.block {
+    border-left: 2px solid #e8eced;
+    margin-left: 67px;
+    padding-left: 55px;
+    padding-top: 202px;
+    max-width: 810px;
+}
+
+.intro__suptitle {
+    width: 100%;
+    color: #b4b4b4;
+    font-family: "Proxima Nova Rg";
+    font-size: 20px;
+    font-weight: 400;
+    font-style: normal;
+    line-height: normal;
+    text-align: left;
+    text-transform: uppercase;
+    letter-spacing: 6.72px;
+    margin-bottom: 47px;
+}
+
+.intro__title {
+    width: 100%;
+    color: #2a2a2a;
+	font-family: 'Futura PT';
+    font-size: 78px;
+    font-weight: 800;
+    font-style: normal;
+    text-align: left;
+    text-transform: uppercase;
+    letter-spacing: 5.28px;
+    line-height: normal;
+}
+
+.intro__entitlу {
+    width: 100%;
+    color: #8f8f8f;
+    font-family: "Georgia";
+    font-size: 30px;
+    font-style: italic;
+    letter-spacing: normal;
+    text-align: justify;
+    line-height: normal;
+    margin-bottom: 55px;
+}
+
+.intro__text {
+    width: 100%;
+    color: #000000;
+    font-family: Georgia;
```

Comment:
"Georgia" or Georgia?
You need to unify it. Better to use with brackets (`"Georgia"`)